.gitignore

```diff
@@ -1,3 +1,5 @@
 .*.cache
 composer.lock
+.phpunit.result.cache
+.phpunit.cache
```

Comment:
these two cases are already covered above. Please remove

Reply:
thanks, i did not see this. reverted